components/AvatarList/index.js

```diff
@@ -18,8 +20,16 @@ const AvatarList = () => {
         </div>
       </div>
       <div className={styles.buttonContainer}>
-        <button>Cancel</button>
-        <button>Save</button>
+        <Link href="/settings">
+          <a>
+            <button>Cancel</button>
```

Comment:
Nice work using the Next.js <Link> component!

We can remove the button element altogether.

Styling can be applied by add a class to the anchor tag.

Reply:
*I know this is merged. Sharing for reference moving forward.

Reply:
Thanks Joel.

I think I might create a LinkedButton component to be reused throughout the application, correctly omitting the button element and styling the anchor tag appropriately.

For corrections like this that are required app-wide, should we make the changes in each individual branch thus creating several new PRs? Or can I make a new ticket/branch addressing the creation of the new reusable component and the corrections needed?